engines/xeen/party.cpp

```diff
@@ -812,6 +820,18 @@ bool Party::arePacksFull() const {
 	return total == (_activeParty.size() * NUM_ITEM_CATEGORIES);
 }
 
+const char *Party::getFoundForm(const Character &c) {
+	if (Common::RU_RUS == g_vm->getLanguage()) {
```

Comment:
it could be more readable with
```
if ((Common::RU_RUS == g_vm->getLanguage()) &&
    (c._sex == FEMALE)){
    return Res.FOUND[1];
} else {
    return Res.FOUND[0];
}
```

Reply:
Fixed